client/src/js/services/reference.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * @description 
+ *
+ * @returns 
+ */
+
+angular.module('bhima.services')
+  .factory('reference', reference);
+  
+reference.$inject = ['$http'];
+
+function reference($http) {  
+  
+  function list() { 
+    var path = '/references/default';
```

Comment:
Why is this route `/references/default`?  Why not `/references/detailed` or just `/references`?